pkg/catalog/routes.go

```diff
@@ -538,13 +586,6 @@ func (mc *MeshCatalog) getDestinationServicesFromTrafficTarget(t *access.Traffic
 func (mc *MeshCatalog) buildInboundPolicies(t *access.TrafficTarget, svc service.MeshService) []*trafficpolicy.InboundTrafficPolicy {
 	inboundPolicies := []*trafficpolicy.InboundTrafficPolicy{}
 
-	// fetch services running workloads with destination service account
-	destServices, err := mc.getDestinationServicesFromTrafficTarget(t)
-	if err != nil {
-		log.Error().Err(err).Msgf("Error resolving destination from traffic target %s (%s)", t.Name, t.Namespace)
-		return inboundPolicies
-	}
-
 	// fetch all routes referenced in traffic target
```

Comment:
destination services has been removed as discussed in a previous PR : https://github.com/openservicemesh/osm/pull/2593#discussion_r581304432